errbot/config-template.py

```diff
@@ -93,6 +93,12 @@
 # For example CORE_PLUGINS = ('ACLs', 'Backup', 'Help') you get those names from the .plug files Name entry.
 # For absolutely no plug: CORE_PLUGINS = ()
 
+# Defines an order in which the plugins are getting their callbacks. Useful if you want to have plugins do
+# pre- or post-processing on messages.
+# The 'None' tuple entry represents all the plugins that aren't to be expicitely ordered. For example, if
```

Comment:
not really, if you don't have the None parameter, it is just to say "Enforce the callback order between those plugins in that order". If another plugin interferes by its order, you'll need to add it to the list. As I understand it the usual case (99.9%?) is that we don't care about this order and there is a couple rare cases where you know 2 plugins that need to be called in a specific order. Do you have any counter examples or stuff I miss maybe ?